front/tests/pages/study-page-model.ts

```diff
@@ -83,6 +89,8 @@ class StudyPage {
     this.getStudyBudgetInput = page.getByLabel('Budget');
     this.getStudyUpdateConfirmBtn = page.locator('#modal-content').getByTestId('updateStudy');
     this.getStudyDeleteConfirmBtn = page.locator('#modal-content').getByTestId('deleteStudy');
+    this.createStudyButton = page.getByTestId('createStudy');
```

Comment:
Can you rename the test-id "create-study" to harmonize with the other test-ids ?